src/django_peerctl/models/peerctl.py

```diff
@@ -342,6 +342,8 @@ def get_or_create(cls, asn, org):
             if not obj.org and org:
                 obj.org = org
                 obj.create_global_policy()
+            elif obj.org and org and (obj.org != org):
+                raise IntegrityError(f"ASN {asn} already claimed by another org")
```

Comment:
@jason20c i think that can just raise `ASNClaimed` directly - Integrity error seems database internal